e2e/flows/set-default-owner-and-scope.e2e.4.ts

```diff
@@ -0,0 +1,116 @@
+      componentId = `${fullScope}/utils/is-type`;
+      componentPackageName = `@${defaultOwner}/${defaultScope}.utils.is-type`;
+      helper.bitJsonc.addDefaultOwner(defaultOwner);
+      helper.bitJsonc.addDefaultScope(defaultScope);
+      helper.fixtures.populateWorkspaceWithUtilsIsType();
+      const rawLinkOutput = helper.command.link('-j');
+      parsedLinkOutput = JSON.parse(rawLinkOutput);
+      helper.command.tagAllComponents();
+    });
+    it('should create link with default owner as prefix', () => {
+      const linkFolderPath = path.normalize(`node_modules/${componentPackageName}`);
+      const linkFullPath = path.join(linkFolderPath, '/utils/is-type.js');
+      const outputLinkPath = parsedLinkOutput.linksResults[0].bound[0].to;
+      expect(outputLinkPath).to.equal(linkFullPath);
+      expect(path.join(helper.scopes.localPath, 'node_modules')).to.be.a.directory();
+      expect(path.join(helper.scopes.localPath, linkFolderPath)).to.be.a.directory();
+    });
+    describe('after export', () => {
+      let exportOutput;
+      before(() => {
+        exportOutput = helper.command.exportAllComponents();
+      });
+      it('should export the component to correct scope', () => {
+        expect(exportOutput).to.have.string('exported 1 components');
+        expect(exportOutput).to.have.string(`${defaultOwner}.${defaultScope}`);
+      });
+      describe('validate models data', () => {
+        let compModel;
+        let versionModel;
+        before(() => {
+          compModel = helper.command.catComponent(`utils/is-type`);
+          versionModel = helper.command.catComponent(`utils/is-type@latest`);
+        });
+        it('should store scope name with the format owner.scope in the models', () => {
+          expect(compModel.scope).to.equal(`${fullScope}`);
+        });
+        it('should store scope name with the format owner.scope in the models', () => {
+          expect(compModel.bindingPrefix).to.equal(`@${defaultOwner}`);
+          expect(versionModel.bindingPrefix).to.equal(`@${defaultOwner}`);
+        });
+      });
+      describe('on imported', () => {
+        before(() => {
+          helper.scopeHelper.reInitLocalScope();
+          helper.scopeHelper.addRemoteScope();
+          helper.command.importComponent('utils/is-type');
+        });
+        it('should create links for the imported components', () => {
+          const linkFolderPath = path.normalize(`node_modules/${componentPackageName}`);
+          expect(path.join(helper.scopes.localPath, 'node_modules')).to.be.a.directory();
+          expect(path.join(helper.scopes.localPath, linkFolderPath)).to.be.a.directory();
+        });
+        describe('pack component', () => {
+          let parsedJson;
+          before(() => {
+            const componentIdWithVersion = `${componentId}@0.0.1`;
+            const packDir = path.join(helper.scopes.localPath, 'pack');
+            const options = {
```

Comment:
hua?

Reply:
@qballer Not sure what you are trying to say

Reply:
Same here. . .. Why do you have the options object?

Reply:
I need to pass it to the pack command itself.